tools/test/go.mod

```diff
@@ -1,10 +1,20 @@
 module github.com/stackrox/stackrox/tools/test
 
-go 1.21
+go 1.22.0
 
-toolchain go1.21.7
+toolchain go1.22.1
```

Comment:
Are we there yet?

Reply:
Not sure what you mean.. This was changed automatically by `go mod tidy` AFAICT. These days go seems to auto-download any needed toolchain on the fly.

Reply:
In general yes but not on downstream builds 
- https://github.com/stackrox/stackrox/pull/11088

Reply:
I don't think downstream build would look at this file, but I reverted this bit just in case, and it seems to build fine. I guess it's goland being overzealous.